### ISSUE TITLE
Redirect and error handling of deleted/unpublished pages

### ISSUE THREAD

**Issue by c00n84** [sulu/sulu#3102]
| Q | A
| --- | ---
| Bug? | no
| New Feature? | yes
| Sulu Version | -
| Browser Version | -

#### Actual Behavior

Currently I get a Error 404 page when I delete a page or put it in status "test".

#### Expected Behavior

When I delete a page or put it in status "test", I want to decide what happens with the url. I can choose between Error 404 or to redirect the url to another page of my webspace.

#### Possible Solutions

I'll get an overlay where I can choose between my solutions.

**Comment by dvdlwsk** [comment]
Good idea @c00n84. We deal with the same problem and it would be create to set redirects for unpublished sites.

**Comment by alexander-schranz** [comment]
@dvdlwsk maybe the [SuluRedirectBundle](https://github.com/sulu/suluredirectbundle) will solve it currently for you. It will not provide the overlay feature but the content manager is able to add redirects in a list.
